src/machine/machine_mimxrt1062_i2c.go

```diff
@@ -171,6 +171,12 @@ func (i2c *I2C) Configure(config I2CConfig) {
 	i2c.reset(freq)
 }
 
+// SetBaudRate sets the communication speed for I2C.
+func (i2c I2C) SetBaudRate(br uint32) error {
+	// TODO: implement
+	return nil
```

Comment:
Hmmmm... shouldn this return an error or panic until it is implemented?

Reply:
Good idea. Done.